.github/workflows/benchmark.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         git clone https://github.com/Chia-Network/clvm_tools.git --branch=main --single-branch
         python -m pip install ./clvm_tools
         python -m pip install colorama
-        maturin develop --release
+        maturin develop --release --cargo-extra-args="--features extension-module,openssl"
```

Comment:
what does this do?

Reply:
I'm splitting out options like python API and OpenSSL support into "feature" flags. See also <https://doc.rust-lang.org/cargo/reference/features.html>. Seems to be the cargo way of doing it (rather than Unix vs Windows, which leaves webasm out in the cold).

Reply:
it seems like quite inconvenient to have to type all this to build. Is there a good reason not to enable these features by default?

Reply:
I expect the end-game is to have separate projects (maybe even in separate repos) for the core `clvm_rs`, the python bindings and the wasm bindings. At that point `extension-module` wouldn't be a feature anymore, at least.

I would think `openssl` could have a default that depends on the operating system too, at least for now.